ChangeLog

```diff
@@ -1,3 +1,6 @@
+11.0.0:
+ - Removed support for v4.
```

Comment:
Make sure you update this in a followup commit to be more complete. It also looks like you haven't modified version.rb yet. These can both be done in a followup PR after this is merged.